doc/faq/customization.rst

```diff
@@ -130,7 +130,7 @@ do it like this::
 Logo
 ~~~~
 
-The default logo is located at ``privacyidea/static/css/privacyIDEA1.png``.
+The default logo is located at ``privacyidea/static/img/privacyIDEA1.png``.
```

Comment:
I would argue that a logo is part of the style.
Currently we do not have an image directory.
Also this change requires, that all users, who customized their logo (which is actually severals) will have to manually change the location, where they dropped their logo file.
I recommend to not change the logo location!

Reply:
Fixed in c66a29f7